src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let newRobot = {};
+
+  for (const key in robot) {
+    if (!newRobot[robot[key]]) {
+      newRobot[robot[key]] = key;
+    } else {
+      newRobot = null;
+      break;
```

Comment:
swap if branches, because its better to have condition without not if you have else